airflow/providers/celery/executors/celery_executor.py

```diff
@@ -469,7 +469,7 @@ def cleanup_stuck_queued_tasks(self, tis: list[TaskInstance]) -> list[str]:
         for ti in tis:
             readable_tis.append(repr(ti))
             task_instance_key = ti.key
-            self.fail(task_instance_key, None)
+            self.failed_in_queue(task_instance_key, None)
```

Comment:
This one is backward incompatible change with Airflow 2.7, 2.8 and 2.9.
Maybe it is also a good idea to made separate PR in Core and and after in Provider(s)

Reply:
ah i see. let me do that @Taragolis

Reply:
can you explain me the logic behind splitting it into separate PR @Taragolis ?